Bugfix for SQS max_fetch type

jrauen opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Params are passed in as strings but the max_fetch parameter wasn't converted to int before being used.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

JasBeilin commented on Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py:
```diff
@@ -214,7 +214,7 @@ def main():
     retries = params.get('retries') or 5
     aws_queue_url = params.get('queueUrl')
-    max_fetch = min(params.get('max_fetch', 10), 100)
+    max_fetch = min(int(params.get('max_fetch', 10)), 100)
```
```suggestion
    max_fetch = min(arg_to_number(params.get('max_fetch', 10)), 100)
```

JasBeilin commented on Packs/AWS-SQS/ReleaseNotes/1_2_9.md:
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### AWS - SQS
+- Fixed an issue where the `max_fetch` parameter was not parsed into an integer.
```
```suggestion
- Fixed an issue where the *max_fetch* parameter was not parsed into an integer.
```

bakatzir commented:
Hi @jrauen, we could not ignore your multiple contributions to our repository
https://github.com/demisto/content/pulls?q=is%3Apr+author%3Ajrauen
And wanted to send you some Cortex/PANW swag, as a small token of appreciation.
can you share your email/DFIR(https://start.paloaltonetworks.com/join-our-slack-community) so we can coordinate this?

jrauen commented:
DFIR: badarsebard

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=19544) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=19544) before we can accept your contribution.<br/>**20** out of **22** committers have signed the CLA.<br/><br/>:white_check_mark: jrauen<br/>:white_check_mark: content-bot<br/>:white_check_mark: ShirleyDenkberg<br/>:white_check_mark: amshamah419<br/>:white_check_mark: JudahSchwartz<br/>:white_check_mark: dorschw<br/>:white_check_mark: GuyAfik<br/>:white_check_mark: Ni-Knight<br/>:white_check_mark: Shellyber<br/>:white_check_mark: barchen1<br/>:white_check_mark: MosheEichler<br/>:white_check_mark: DeanArbel<br/>:white_check_mark: evisochek<br/>:white_check_mark: merit-maita<br/>:white_check_mark: ilaner<br/>:white_check_mark: yaakovi<br/>:white_check_mark: daryakoval<br/>:white_check_mark: dansterenson<br/>:white_check_mark: ilappe<br/>:white_check_mark: idovandijk<br/>:x: guytamir10<br/>:x: tomer-pan<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=19544) it.</sub>

jrauen commented:
> Hi @jrauen, Thank you for your contribution! Please check my suggestions. Also note that you have not updated the integration's version in the metadata file. Please fix so we can move forward :)

Apologies for the confusing previous commit, I had rebased my branch onto master out of habit and it cluttered things up. I've reverted to the original parent commit and things look clean. You can verify the changes you've requested in the latest commit.
